OrbitCore/CaptureData.h

```diff
@@ -29,10 +32,24 @@ class CaptureData {
     return selected_functions_;
   }
 
+  [[nodiscard]] int32_t process_id() { return process_id_; }
+
+  [[nodiscard]] const std::string& process_name() { return process_name_; }
+
+  [[nodiscard]] const std::chrono::system_clock::time_point&
+  capture_time_point() {
```

Comment:
let's take this opportunity to rename this to capture_start_time

Reply:
done.